src/Repositories/CustomUserRepository.php

```diff
@@ -74,44 +62,13 @@ public function updateOrCreate($data): object
     }
 
     /**
-     * Authenticate a user with a decoded ID Token
-     *
-     * @param array $decodedJwt
-     *
-     * @return Auth0JWTUser
-     */
-    public function getUserByDecodedJWT(array $decodedJwt): Auth0JWTUser
-    {
-        $user = $this->upsertUser((array) $decodedJwt);
-
-        return new Auth0JWTUser($user->getAttributes());
-    }
-
-    /**
-     * Authenticate a user with a decoded ID Token
-     *
-     * @param object $jwt
-     *
-     * @return int user ID
-     */
-    public function getUserIDByDecodedJWT($jwt): int
-    {
-        $user = $this->upsertUser((array) $jwt);
-
-        return $user->id;
-    }
-
-    /**
-     * Get a User from the database using Auth0 profile information
-     *
-     * @param array $userinfo
+     * @inheritdoc
+     * This method is not in use.
      *
-     * @return Auth0User
      */
-    public function getUserByUserInfo(array $userinfo): Auth0User
-    {
-        $user = $this->upsertUser($userinfo['profile']);
-
-        return new Auth0User($user->getAttributes(), $userinfo['accessToken']);
+    public function fromAccessToken(
+        array $user
+    ): ?\Illuminate\Contracts\Auth\Authenticatable {
+        return null;
```

Comment:
An implementation for this method does not seem to be necessary for the purposes of this library.